newapi/ooniapi/probe_services.py

```diff
@@ -142,6 +165,17 @@ def check_in():
     run_type = data.get("run_type", "timed")
     charging = data.get("charging", True)
 
+    resp = dict(
+        v=1
+    )
+
+    if probe_cc == "ZZ" and asn == "AS0":
```

Comment:
Do we want to return CC and ASN in every case instead?

Reply:
I was originally thinking about that, however since probe_cc and probe_asn are user supplied input I think we should ensure that validation of it is very strict.
In order to avoid that I figured we would return it only in cases where it's needed, as otherwise it's basically just echoing back the same data the user supplied.

Or is your suggestion to always do the lookup, even when the user provides probe_cc and probe_asn? In that case, which value should have priority when performing the test list lookup?